examples/scheduled-actions-workspace/README.md

```diff
@@ -0,0 +1,12 @@
+# Scheduled Actions Example
+
+Executes scheduled actions defined in the projects .vscode/settings.json files and in the workspace scheduled.code-workspace file.
+
```

Comment:
Consider to get here more story related description rather then technical - Paola review needed ....

Reply:
I will merge it now and will work with Paola only on this file. Thanks